app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    not_vaccinated = False
+    not_wearing_masks = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            not_vaccinated = True
+        except NotWearingMaskError:
+            not_wearing_masks += 1
+
+    if not_vaccinated:
+        return "All friends should be vaccinated"
```

Comment:
The message 'All friends should be vaccinated' might be misleading if only some friends are not vaccinated. Consider rephrasing it to 'Some friends are not vaccinated' for clarity.